src/config/texts.js

```diff
@@ -473,6 +473,11 @@ export const texts = {
       save: 'Speichern',
       sectionHeader: 'Standort'
     },
+    onboarding: {
+      onActivate: 'Beim nächsten Start wird die App-Einführung angezeigt.',
+      onDeactivate: 'Die App-Einführung wird beim nächsten Start nicht angezeigt.',
+      ok: 'OK'
```

Comment:
for the `dateTimePicker` around line 65 we have `ok: 'Ok'`, so i would suggest to do it the same way here for consistency.

Reply:
I wanted to add it again for code readability. I will use this.

Reply:
should it stay like this for now? the same is true for matomo setting.

Reply:
a7587b4e5e398ac83c0cd01a398797166b4d72b2